### PR TITLE
docs, multi-homing: secondary attachment config via net-selection-elements

### PR DESCRIPTION
**Pull request by maiqueb**
**- What this PR does and why is it needed**

This commit adds a section to the user documentation explaining how to configure the following data for a pod's attachment:
- IP address
- MAC address
- pod interface name

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: docs/multi-homing.md [hunk 1]
```diff
@@ -191,6 +191,47 @@ spec:
...
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "l2-network",
+        "mac": "02:03:04:05:06:07",
+        "interface": "myiface1",
+        "ips": [
+          "192.168.200.20/24"
```
Comment (dougbtv) [inline docs/multi-homing.md:214]: nit: use the doc range `192.0.2.0/24` etc

**Review by dougbtv — APPROVED** [review]

File: docs/multi-homing.md (hunk 1 above)
Comment (maiqueb) [reply docs/multi-homing.md]: Done.

**Comment by cathy-zhou** [comment]
lgtm

**Pull request merged**